src/components/apps/dungeon/home.tsx

```diff
@@ -975,7 +975,7 @@ export function DungeonApp()
                     setEnemyState(DungeonStatus.dead);
         
                     //Victory sound plays
-
+                    VictoryAudio.volume=volume/100
```

Comment:
is there anywhere you don't do volume / 100?  can you not just do this before it is passed on